CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [30.0.0] - 2022-03-24
```

Comment:
Delete these changes and redo with minor version bump.

Reply:
Done